libs/gi/page-team/src/CharacterDisplay/Tabs/TabTheorycraft/ArtifactMainStatAndSetEditor/ArtifactMainLevelSlot.tsx

```diff
@@ -86,7 +86,7 @@ export function ArtifactMainLevelSlot({
           </Box>
         ) : (
           <DropdownButton
-            sx={{ px: 0 }}
+            sx={{ px: 0, padding: '0.5rem' }}
```

Comment:
sort of a nit, but should be 
```suggestion
            sx={{ px: '0.5rem' }}
```